packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json

```diff
@@ -7,6 +7,8 @@
   "credentials_success_right_pane_button_caption": "Ga naar {{verifierUrlCaption}}",
   "credentials_success_right_pane_paragraph": "Uw {{credentialName}} credential is met succes aan u uitgereikt.\n\nU kunt het gebruiken om uw identiteit te verifiëren wanneer u interactie heeft met andere partijen.",
   "credentials_success_right_pane_title": "Gefeliciteerd!",
+  "credentials_success_right_pane_verified_account_paragraph": "\r\n\n\nThe Verified {{credentialName}} credential proofs that UnifiedPost\nhas been able to verify that you have access to the  e-mail address you provided.\r\n\nYou can use this credential to Login to your UnifiedPost account.",
```

Comment:
result is that it now says: Verified Account credential credential

Reply:
Just followed the pattern that was already in the translation files and in the legal-entity config file, but they seem to be wrong. Fixed following the code from Niels.